internal/testrunner/runners/system/servicedeployer/terraform_env.go

```diff
@@ -7,14 +7,17 @@ package servicedeployer
 import (
 	"errors"
 	"fmt"
+	"os"
 	"strings"
 
 	"github.com/elastic/elastic-package/internal/compose"
 )
 
 const (
 	tfDir       = "TF_DIR"
+	tfOutputDir = "TF_OUTPUT_DIR"
 	tfTestRunID = "TF_VAR_TEST_RUN_ID"
+	outputDir   = "/tmp/"
```

Comment:
Prefer `os.TempDir()` to avoid any OS specific paths.

Reply:
@andrewkroh This seems to be an issue in CI. The buildkite user does not have permissions to create new directories in `os.TempDir()`. Using `build` directory instead?

Reply:
Changed this as per suggestion [here](https://github.com/elastic/elastic-package/pull/1272#discussion_r1211694536)